project/sites/ircommission/themes/ircommission_theme/src/scss/components.scss

```diff
@@ -617,3 +614,11 @@ footer {
     }
   }
 }
+
+.current-search-item-results-count {
+  text-align: left;
+}
+
+article header h2 {
+  text-align: left;
```

Comment:
Unity theme should take care of search count and article styling, you shouldn't need to change it. Maybe missing a twig template?

Reply:
Thanks I have now removed this